client/src/components/CreateBoard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { setCurrentBoard, setBoards } from '../redux/actionCreators.js';
+import { connect } from 'react-redux';
+
+const CreateBoard = (props) => {
+  return (
+    <div>This is CreateBoard</div>
+  );
+};
+
+const mapStateToProps = (state) => {
+  return {
+    'boards': state.boards,
+    'currentBoard': state.currentBoard
+  };
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    handleSetCurrentBoard(boardClicked) {
```

Comment:
How does the handleSetCurrentBoard get injected into this context? I only see import of setCurrentBoard and setBoards. Is there some magic here?